bin/options.js

```diff
@@ -22,9 +22,10 @@ function getOptions() {
   try {
     var opts = fs.readFileSync(optsPath, 'utf8')
           .trim()
+          .replace(/\\\s+/, '%20')
           .split(/\s+/)
-          .filter(function(value) {
-            return value ? true : false;
```

Comment:
I agree. I've changed these lines a bit to simplify them, see https://github.com/mochajs/mocha/compare/630156d35a19bf5dfe722899084c3f7edaee764a...9a340570dfc5d56d7e8cda1a6046a8f169e85e31